lib/page.rb

```diff
@@ -130,7 +134,8 @@ def click_button(button_name, button_index: nil)
       end
       begin
         button.click
-      rescue Exception
+      rescue Exception => exception
+        Log.debug("page.click_button exception was rescued: #{exception}")
```

Comment:
This seems a *little* redundant with the next line. Can we merge the two debug statements so we don't print two lines of debug output for this exception?